From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matthias Maennich <maennich@google.com>
Date: Tue, 9 Jun 2020 13:38:27 +0200
Subject: abipkgdiff: remove unused includes of elfutils/libdw.h and elf.h

Definitions from those headers are not actually used. Yet the includes
create unnecessary dependencies.

	* tools/abipkgdiff.cc: Drop unused includes.

Signed-off-by: Matthias Maennich <maennich@google.com>
---
 tools/abipkgdiff.cc | 2 --
 1 file changed, 2 deletions(-)

diff --git a/tools/abipkgdiff.cc b/tools/abipkgdiff.cc
index 7d946e72b3c8..445e2406830b 100644
--- a/tools/abipkgdiff.cc
+++ b/tools/abipkgdiff.cc
@@ -72,8 +72,6 @@
 #include <assert.h>
 #include <fcntl.h>
 #include <sys/stat.h>
-#include <elf.h>
-#include <elfutils/libdw.h>
 
 // If fts.h is included before config.h, its indirect inclusions may
 // not give us the right LFS aliases of these functions, so map them
